vassal-app/src/main/java/VASSAL/configure/BeanShellExpressionConfigurer.java

```diff
@@ -210,7 +210,7 @@ public Component getControls() {
       validator = new Validator();
       nameField = new JTextArea(1, 100);
 
-      nameField.setFont(new Font("Monospaced", Font.PLAIN, 14));
+      nameField.setFont(new Font("courier new", Font.PLAIN, 14));
```

Comment:
"courier new" isn't guaranteed to exist. What happens when it doesn't?

Reply:
Also, why have this differ from the monospaced fonts everywhere else?

Reply:
oops, that was a test to confirm that on my system, Courier New was in fact the selected monospaced on windows. Fixed.